pkg/config/config_test.go

```diff
@@ -829,20 +828,20 @@ periodics:
 	for _, tc := range testCases {
 		t.Run(tc.name, func(t *testing.T) {
 			// save the config
-			prowConfigDir, err := ioutil.TempDir("", "prowConfig")
+			prowConfigDir, err := os.MkdirTemp("", "prowConfig")
```

Comment:
For tests, its better to use: https://pkg.go.dev/testing#T.TempDir

Reply:
Updated.